doc/index.rst

```diff
@@ -50,26 +50,26 @@ Learning resources
     Tutorials
     ^^^^^^^^^
 
-    - :doc:`Quick-start Guide <tutorials/introductory/usage>`
-    - :doc:`Plot Types <plot_types/index>`
-    - `Introductory Tutorials <../tutorials/index.html#introductory>`_
-    - :doc:`External Learning Resources <resources/index>`
+    - :doc:`Quick-start guide <tutorials/introductory/usage>`
+    - :doc:`Plot types <plot_types/index>`
+    - `Introductory tutorials <../tutorials/index.html#introductory>`_
+    - :doc:`External learning resources <resources/index>`
 
     ---
 
     How-tos
     ^^^^^^^
-    - :doc:`Example Gallery <gallery/index>`
+    - :doc:`Example gallery <gallery/index>`
     - :doc:`Matplotlib FAQ <faq/index>`
 
     ---
 
     Understand how Matplotlib works
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    - The :ref:`users-guide-explain` section of the :doc:`Users guide <users/index>`
+    - The :ref:`users-guide-explain` in the :doc:`Users guide <users/index>`
```

Comment:
Can we just use the 'Usage guide' title instead of renaming it to Users guide?

Reply:
Done.